nicolegooden/indecision-maker
# Refactor

#### What's this PR do?  

+ Removes white space from files
+ Changes style for activity cards displayed on the `BrowsePage` so that only the title shows up for each card game, but styles for all other activity types remain unchanged
+ Adds `prop types` for any component that is receiving props in App's render.  I checked that other components rendered outside of App are accounted for as well (none had props)
+ Removes `About` from top of Homepage
+ Removes functions from inside of functional components; they are now variables with the same purpose
+ Changes styles for the hover state of form's choices - on hover, the user will now see the choice pop out (increase in text size) and the text shadow has been reduced
+ Deploys this repo via GH pages

#### Where should the reviewer start?

These changes exist in most of the files, feel free to just view the files in order.  Most changes happen at the bottom of the file or in the file's `render()`. 

#### How should this be manually tested?  

Pull down this version of code and ensure all functionality is in tact (mostly referring to form here).  Hover over the choices in the `Form` component to see the differences in styling.  In the browser, ensure there are no errors for failed prop types.

#### Any background context you want to provide?  

N/A

#### What are the relevant tickets?  

Closes #64 
Closes #65
